aries_vcx/Cargo.toml

```diff
@@ -50,6 +50,7 @@ derive_builder = "0.10.2"
 tokio = { version = "1.20.4" }
 thiserror = "1.0.37"
 url = { version = "2.3", features = ["serde"] }
+async-lock = "2.7.0"
```

Comment:
Why introduce a new crate instead of using `tokio::sync::RwLock`, since we already depend on `tokio`.

Reply:
removed, not needed after all